pandas-stubs/core/dtypes/missing.pyi

```diff
@@ -34,6 +43,8 @@ def notna(obj: Series) -> Series[bool]: ...
 def notna(obj: Index | list | ArrayLike) -> np.ndarray: ...
```

Comment:
Should this also be npt.NDArray[np.bool_]

Reply:
agreed.  Good catch

Reply:
You're definitely right \o/